lib/ex_ami/client.ex

```diff
@@ -23,7 +23,8 @@ defmodule ExAmi.Client do
   # API
 
   def child_spec([_, worker_name | _] = args) do
-    IO.inspect args, label: "ExAmi.child_spec 1"
+    IO.inspect(args, label: "ExAmi.child_spec 1")
```

Comment:
I don't think we should leave IO.inspect statements around. Convert this and others to Logger.info.

Reply:
Agree, remove it.

Reply:
Removed them.